tox.ini

```diff
@@ -1,7 +1,9 @@
 [tox]
 envlist = py27, py35
```

Comment:
I love the future :) but just asking this because it force us to run newer that Ubuntu 16.04 to have the official python package installed. This doesn't run in staging nor production environment so I guess this is ok anyway.